src/fragments/lib-v1/storage/js/configureaccess.mdx

```diff
@@ -0,0 +1,92 @@
+
+* Guest: Accessible by all users of your app. Files are stored under the `public/` path in your S3 bucket.
+* Protected: Readable by all users, but writable only by the creating user. Files are stored under `protected/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.
+* Private: Only accessible for the individual user. Files are stored under `private/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.
+
+When using Auth and Storage modules together, you do not need to construct the `/{user_identity_id}/` manually as the library will use the configured Cognito Identity ID for your user/device along with the configured access level for an action. This includes UnAuthenticated access where you will first call `Auth.currentCredentials()` before a Storage action. See [Authentication](/lib/auth/overview) for more information.
+
+The access level can be configured on the Storage object globally. Alternatively, the access levels can be set in individual function calls.
+
+> Default access level for Storage module is `guest`. Unless you configure Storage otherwise, all uploaded files will be publicly available for all users.
+
+Access level configuration on the Storage object:
+
+```javascript
+import { getUrl } from 'aws-amplify/storage'
+
+await getUrl({ key : 'welcome.png', options: { level: 'guest' }}); // Gets welcome.png in public space
+```
+
+The default access level is `guest`:
+```javascript
+import { getUrl } from 'aws-amplify/storage'
+
+await getUrl({ key: 'welcome.png' }); // Get welcome.png in public space
+```
+
+## Customization 
+
+### Customize Object Key Path 
+
+You can customize your key path by defining a prefix resolver:
+
+```javascript
+Amplify.configure(
+    {},
+    {
+        Storage: {
+            S3: {
+                prefixResolver: async ({ accessLevel, targetIdentityId }) => {
+                    if (accessLevel === 'guest') {
+                        return 'myPublicPrefix/';
+                    } else if (accessLevel === 'protected') {
+                        return `myProtectedPrefix/${targetIdentityId}`;
+                    } else {
+                        return `myPrivatePrefix/${targetIdentityId}`;
+                    }
+                },
+            },
+        },
+    }
+);
+
+```
+
+For example, if you want to enable read, write and delete operation for all the objects under path *myPublicPrefix/*,  declare it in your IAM policy:
+
+```xml
```

Comment:
nit: use complete json snippet as example
and the syntax highlight should be `json`